packages/client/src/solanaDepositAndCall.ts

```diff
@@ -88,9 +89,25 @@ export const solanaDepositAndCall = async function (
       );
     }
 
+    const valuesArray = args.params[1].map((value: any, index: any) => {
```

Comment:
Since `args.params` is of type `any` there's no way to know for sure if `args.params[1]` is defined, we should probably check that with something like `args.params[1]?.map` and stop executing instructions if `valuesArray` results in `undefined`.